scanner/src/main/kotlin/scanners/ScanCode.kt

```diff
@@ -283,8 +283,25 @@ class ScanCode(name: String, config: ScannerConfiguration) : LocalScanner(name,
         }
     }
 
+    private fun getFileCount(result: JsonNode): Int {
+        result.get("headers")?.forEach { header ->
```

Comment:
Thinking about it again, wouldn't it be best to not read the 'headers' node at all?
It is just performance optimization which is complete non sense here isn't it ?